### DISCUSSION TITLE
PO-TABLE - Não apresentação de ícone com type: 'icon'

### DISCUSSION THREAD

**Discussion by ThiagoLeite06** [po-ui/po-angular#834] [Tire suas dúvidas] [answered]
Recentemente fiz algumas atualizações no projeto que estou atuando, de Angular 8 > 11 e Portinari > PO-UI.

Porém, em algumas tabelas estou com problemas nos ícones.
Estou seguindo a documentação de tabelas onde mostra o exemplo de 2 ícones dentro da célula. No meu projeto, quando coloco type: 'icon' dentro do array de PoTableColumn, ele não me traz nada.

Os dados vindos da minha API estão com os mesmos nomes dos property. Se puxo um id, por exemplo, ele só aparece se não tiver com o type. Minha necessidade é colocar um ícone na célula e que o tooltip seja esse id.

Gostaria de saber se alguém consegue me ajudar, pois não encontro erros em console nem nada. Não sei se não estou sabendo fazer ou se é algum efeito colateral da minha atualização.

Obrigado.

**Comment by jhonyeduardo** [comment, marked as answer]
Bom dia,

Para exibir os ícones, atualmente, você precisa retornar nos items da tabela os valores correspondente ao ícone que deseja exibir, por exemplo:
// colunas
```
[
{ property: 'columnIcon', label: 'Actions', type: 'icon', icons: [
  { value: 'add', icon: 'po-icon-plus', color: 'color-06', action: this.add.bind(this), tooltip: 'Adiciona um novo item' },
  { value: 'edit', icon: 'po-icon-edit', action: this.edit.bind(this) },
  { value: 'delete', icon: 'po-icon-delete', color: 'color-12', action: this.remove.bind(this) }
]},
]
```
// registros
```
[{ id: 123, name: 'John', columnIcon: ['add', 'edit', 'delete'] }]
```

**Reply by viniciusgati** [reply]
valeu cara, isso aí é total nebuloso na doc.
